apps/climatemappedafrica/src/pages/[[...slugs]].js

```diff
@@ -79,12 +79,34 @@ function Page({ blocks = [], fallback }) {
   );
 }
 
+async function fetchWithRetry(context, attempt = 1, maxAttempts = 5) {
+  try {
+    const result = await getPageStaticProps(context);
+    return result;
+  } catch (error) {
```

Comment:
🤔 ... isn't it better to retry a failing fetch call (I'm assuming probably the one fetching a geometry) rather than retrying the whole page build?